bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoLightHandler.java

```diff
@@ -179,6 +179,7 @@ private synchronized WemoBridgeHandler getWemoBridgeHandler() {
         return this.wemoBridgeHandler;
     }
 
+    @SuppressWarnings("null")
```

Comment:
Did you check why you have all those warnings? Isn't there a way to change the code, so that those warnings do not happen?

Reply:
done, code changed